backend/clubs/serializers.py

```diff
@@ -1136,6 +1136,20 @@ def validate_target_years(self, value):
         Ensure that the user does not select all of the target years.
         """
         if len(value) >= Year.objects.count():
+            if settings.BRANDING == "fyh":
+                raise serializers.ValidationError(
+                    "We noticed you selected all of the options\
+                     for one or more of the previous tags."
+                    "In order to best optimize our sorting algorithm,\
+                     you need to select only the few \
+                    tags that apply to your resource."
+                    "If you feel that all the tags apply, that's great!\
+                     In that case you would select 'Yes' to the \
+                    question asking if your resource applies to\
+                     all undergraduate, graduate, and professional Penn students."
+                    "Thanks for doing your part to ensure that Hub@Penn\
+                     quickly and efficiently gets resources to our Penn community. "
+                )
```

Comment:
It would be better to use the:

```
abc = (
    "This is a sentence! "
    "This is another sentence!"
)
```

This multiline string formation doesn't result in a bunch of whitespace between the text. Although the browser will collapse the text while rendering it, it'll be nice to fix this at the server level.

```python
>>> abc = (
...     "This is a sentence! "
...     "This is another sentence!"
... )
>>> abc
'This is a sentence! This is another sentence!'
>>> abc = (
...     "This is a sentence!\
...      This is another sentence!"
... )
>>> abc
'This is a sentence!     This is another sentence!'

```